.dockerignore

```diff
@@ -0,0 +1,11 @@
+# Ignore secrets
+.env
+docker-compose.yml
+
+# Ignore git repo
+.git*
+
+# Ignore temporary files and logs
+tmp/*
+log/*
+.byebug_history
```

Comment:
We need to add 
```
mysql
```
to avoid copying db data back into app container.

Reply:
done